bin/build_config

```diff
@@ -8,8 +8,8 @@ require 'rubocop-rspec'
 require 'rubocop/rspec/description_extractor'
 require 'rubocop/rspec/config_formatter'
 
-cops = '{*.rb,capybara/*.rb,factory_bot/*.rb}'
-glob = File.join(__dir__, '..', 'lib', 'rubocop', 'cop', 'rspec', cops)
+glob = File.join(__dir__, '..', 'lib', 'rubocop', 'cop', 'rspec',
```

Comment:
I'm not sure where this would go, but it might be nice to lift this into a constant that can be shared by the spec file as well.

Nice changes either way though 👍

Reply:
I thought the exact same thing: it would nice to share, but where would it go?